pmm-app/src/pmm-qan-app-panel/QueryAnalytics/DataOutput/QueryDetails/Example/Example.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect, useState } from 'react';
+import ExampleService from './Example.service';
+import ReactJson from 'react-json-view';
+
+// const getSQLExample = example => {};
+//
+const getJSONExample = example => (
+  <ReactJson
+    src={example}
+    theme={{
```

Comment:
the theme will be re-created on every render, so it's better to hoist it up, right before the component